sdk/src/types/block/context_input/block_issuance_credit.rs

```diff
@@ -26,6 +26,8 @@ impl BlockIssuanceCreditContextInput {
 }
 
 mod dto {
+    use alloc::format;
```

Comment:
Couldn't this be imported in the macro? 🤔

Reply:
I actually I fixed that In my PR #984 :)